tests/unit/test_base.py

```diff
@@ -78,13 +78,15 @@ class MGR(base.RESTManager):
 
 class TestRESTObject:
     def test_instantiate(self, fake_gitlab, fake_manager):
-        obj = FakeObject(fake_manager, {"foo": "bar"})
+        attrs = {"foo": "bar"}
+        obj = FakeObject(fake_manager, attrs)
 
-        assert {"foo": "bar"} == obj._attrs
+        assert attrs == obj._attrs
```

Comment:
Usually in unit tests it isn't a good idea to do that. Reason being is what if in the method being called it modifies the attrs?

```
def __init__(self, manager, attrs):
   attrs = {"spam": 1}
   self.attrs = attrs
```

`copy` can be used to ensure that the method being tested doesn't modify what gets passed in

```
obj = FakeObject(fake_manager, attrs.copy())
```

Reply:
Thanks! I wanted to do a more complete cleanup of that module so I should have some more presentable PR here soonish 😀